### PR TITLE
docs: add additional linter documentation

### PR DESCRIPTION
**Pull request by SonnevilleDamien**
I added documentation on markdown and docker linter to avoid being rejected by the CI, and modified the package used for unit tests (Unittest -> Pytest) because it's the one we use when running GitHub actions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,14 +86,34 @@ poetry run flake8 --count --show-source --statistics
 poetry run mypy .
 ```
 
+To improve code quality, we use other linter in our workflows, if you don't want to be rejected by the CI,
+please check these additional linter.
```
Comment (ccamel) [inline README.md:90]:
```suggestion
To improve code quality, we use other linters in our workflows, if you don't want to be rejected by the CI,
please check these additional linters.
```

Comment (SonnevilleDamien) [reply README.md]: I fixed it

**Review by ccamel — APPROVED** [review]
LGTM! Thanks for your contribution. 👍

**Review by amimart — APPROVED** [review]
Thanks 👍

**Pull request merged**